### PR TITLE
capture script: option to flip image

### PR DESCRIPTION
**Pull request by Adman**
Signed-off-by: Adrian Matejov <a.matejov@centrum.sk>

### CODE REVIEW

File: scripts/capture.py [hunk 1]
```diff
@@ -49,6 +51,9 @@ def capture_and_save(warmup, threshold, path, image):
         current_date = now.strftime(DIR)
         current_time = now.strftime(FILE)
 
+        if rotate:
+            frame = cv2.flip(frame, -1)
```
Comment (mrshu) [inline scripts/capture.py]: I see what you did there.

What I would really like us to have though is the ability to rotate the input image by a given number of degrees around its center. Just like [here](https://stackoverflow.com/questions/9041681/opencv-python-rotate-image-by-x-degrees-around-specific-point#9042907).

**Review by mrshu — CHANGES_REQUESTED** [review]

**Comment by Adman** [comment]
@mrshu bump

File: scripts/capture.py [hunk 2]
```diff
@@ -8,6 +8,13 @@
 FILE = '%H:%M:%S.png'
 
 
+def rotate_image(image, angle):
+    (h, w) = image.shape[:2]
+    center = (w / 2, h / 2)
```
Comment (mrshu) [inline scripts/capture.py]: @Adman won't this return a float in Python 3 which might cause trouble later? I guess we are not playing that game at all...

File: scripts/capture.py [hunk 3]
```diff
@@ -26,7 +33,9 @@ def matches(frame, threshold, proportion=3):
               help='Path where to save captured images')
 @click.option('--image', default='',
               help='Specify path and get True/False for bright/dark image')
-def capture_and_save(warmup, threshold, path, image):
+@click.option('--rotate', default=0,
+              help='Rotate image (0 to 360 degrees)')
```
Comment (mrshu) [inline scripts/capture.py:37]: @Adman so what exactly happens when I run this script with `--rotate 730`?

**Review by mrshu — CHANGES_REQUESTED** [review]
Just a few small comments, otherwise looks good to me.

File: scripts/capture.py (hunk 3 above)
Comment (Adman) [reply scripts/capture.py:37]: @mrshu The same thing as it would with --rotate 10

Comment (mrshu) [reply scripts/capture.py:37]: @Adman All right, that is fair enough.

File: scripts/capture.py (hunk 2 above)
Comment (Adman) [reply scripts/capture.py]: @mrshu should be fixed

**Review by mrshu — APPROVED** [review]

**Pull request merged**